resources/etc/default/openhab2

```diff
@@ -24,6 +24,11 @@ EXTRA_JAVA_OPTS=""
 #OPENHAB_USERDATA=/var/lib/openhab2
 #OPENHAB_LOGDIR=/var/log/openhab2
 #
+# Set the following variable if you'd like a custom backup location.
+# runtime/bin/backup and runtime/bin/restore will use this path for the zip files.
+#
+#OPENHAB_BACKUPS=/var/lib/openhab2/backups
+#
```

Comment:
I've left it at the top mainly because it's more likely for someone to want to edit it than any of the paths.

OPENHAB_BACKUPS however, would be a very commonly used variable, so perhaps it should be:

```
OPENHAB_BACKUPS
OPENHAB_USER
OPENHAB_GROUP

EXTRA_JAVA_OPTS

OPENHAB_STARTMODE

OPENHAB_*
```